js-browser-sample-app/build.gradle

```diff
@@ -0,0 +1,35 @@
+plugins {
```

Comment:
Let's rename name our folders wit samples in the following way: sample-xxx (both android and this one)